screens/FormScreen.tsx

```diff
@@ -283,7 +283,7 @@ export default connector((props: ConnectedProps<typeof connector> & StackScreenP
       name="InnerFormScreen"
       component={connector(InnerFormScreen)}
       options={{
-        headerTitle: "Telleoversikt",
+        headerTitle: props.route.params.isNewObservation ? "Ny observasjon" : "Tidligere observasjon",
```

Comment:
🐛 Siden dette er en route param og siden FormScreen blir popped og så mounted igjen ved nav til/fra CounterScreen, så bevares ikke verdien, og defaulter til true som definert i App.tsx. Dvs. at ved trykk på tidligere obs. står det først korrekt "Tidligere observasjon", men etter å ha vært innom CounterScreen og så går tilbake så står det "Ny observasjon".

Så jeg tror vi bør flytte dette til redux-state, typ ha "isEditingPreviousObservation" e.l. som en boolean i SauseeState-interfacet

Reply:
I'm on it

Reply:
Fixed